bookkeeper.py

```diff
@@ -33,17 +35,17 @@ def report_reward(self, reward):
         self.reward_sum += reward      # Recall that we must see the outcome of the action
         self.rewards.append(reward)    # before we write down the reward for taking it
     def construct_observation_handler(self):
-        # First, let the observation be the health of both team's Pokemon and also which Pokemon is active.
-        self.state = np.array([0] * 1630)
-        self.shape[-1] = 100
-        self.shape[-2] = 100
-        self.shape[-3] = 100
-        self.shape[-4] = 100
-        self.shape[-5] = 100
-        self.shape[-6] = 100
-        self.shape[-11] = 100
+        self.state = np.array([0] * n)
+        self.state[NUM_POKEMON*2 + TEAM_SIZE + 5] = 100
```

Comment:
Make variable full_health  = 100 and use the variable, so that the code becomes more self-documenting.

Reply:
Done